test/package_config_impl_test.dart

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "package:package_config/package_config_types.dart";
+import "package:test/test.dart";
+import "src/util.dart";
+
+void main() {
+  var unique = Object();
+  var root = Uri.file("/tmp/root/");
+
+  group("LanguageVersion", () {
+    test("minimal", () {
+      var version = LanguageVersion(3, 5);
+      expect(version.major, 3);
+      expect(version.minor, 5);
+    });
+    test("negative major", () {
```

Comment:
nit: be consistent about newlines between tests (a newline between each is generally preferred, but it should be consistent either way)

Reply:
Will try.
VSCode does not help me with its inserted `Run|Debug` lines.

Reply:
Heh ya I have experienced that as well